subworkflows/plink.cwl

```diff
@@ -0,0 +1,65 @@
+cwlVersion: v1.2
+class: Workflow
+id: plink
+doc: |
+  Plink subworkflow. Loads a VCF then runs genome and sexchecks on it.
+requirements:
+- class: ScatterFeatureRequirement
+- class: MultipleInputFeatureRequirement
+- class: InlineJavascriptRequirement
+- class: StepInputExpressionRequirement
+inputs:
+  input_vcf: { type: 'File' }
+  input_ped: { type: 'File?' }
+  output_basename: {type: 'string', doc: "String value to use as basename for outputs"}
+
+  genome: { type: ['null', { type: enum, name: "genome", symbols: ["base", "gz", "rel-check", "full", "unbounded", "nudge"]}], doc: "invokes an IBS/IBD computation. The 'full' modifier adds additional fields. The 'gz' modifier causes the output to be gzipped, while 'rel-check' removes pairs of samples with different FIDs.  The 'unbounded' modifier turns off clipping. Nudge 'nudge' modifier adjusts the final estimates" }
```

Comment:
Can you do like `full` + `gz`? I know STAR has some options where it's limited a limited set to choose from, but you can just list them all if you wanted to

Reply:
Good question. I'll look into it.

Reply:
Ok. Looks like these can be stacked. I'll change the way these are written.